### ISSUE TITLE
Playback Issue

### ISSUE THREAD

**Issue by dany20mh** [dweidenfeld/plexdrive#75]
Hi,

So I tried 2.1.1 and it has playback issue, and it glitches I guess between each chunk.

./plexdrive --chunk-size=20M --clear-chunk-age=10080m -t /Volumes/Backup/plexdrive -v 2 -c ~/.plex --refresh-interval=1m ./mtng

Which based of my setup, it glitch every 15 second or so for a video that I tried.

I compiled also from the source too, but it still have the problem with latest commit.

I revert back to version 2.1.0 and it’s working fine without any problem.

Side note:
My setup is 20M for chunk, when I’m on the 2.1.0, the chunk have exact number, but when I’m on 2.1.1, the chunk have different numbers.
The buffer works perfectly which I can see it’s way ahead of playback, but the video get glitch and jittery every 15 second or so.
Also I deleted the cache file and chunk folder before testing both.

<img width="889" alt="screen shot 2017-05-15 at 11 21 20 am" src="https://cloud.githubusercontent.com/assets/5872728/26065029/b37fb850-3960-11e7-8d1c-7230a9ee4cdc.png">

<img width="879" alt="screen shot 2017-05-15 at 11 24 03 am" src="https://cloud.githubusercontent.com/assets/5872728/26065159/10921c0e-3961-11e7-8480-063adfab421a.png">

**Comment by haljordan2814** [comment]
I'm using default values and playback is working well. What happens when you shift to default values ?

**Comment by dany20mh** [comment]
@haljordan2814 
I would still have problem with the default value, which I only used like 

./plexdrive2 -v 2 -c ~/.plex ./mtng

And because the chunk is 5mb in the default, it was having glitch and jitter each 5 second.

**Comment by dweidenfeld** [comment]
I fixed an issue, that exists since 1.0.0.
With this fix (included in next release) the pre-buffering will be more "aggressive", so you won't get any stuttering while playing back movies, but if you stop a movie it will still load some data in the background and do not stop immediately.

In general: With the new release you will have more load on your network, but there should not be any playback issues anymore.

I have to do some more testing, so if you are using a custom build nightly there still could be bugs in this new logic ;)

**Comment by dany20mh** [comment]
@dweidenfeld 
I tried to build from the latest commit you released few hours ago, and I still have the problem.

1. Buffer is very bad now, as of now 2.1.0 will take maximum 7 second to buffer the new video and start it, but with the latest commit, it will take minimum 16 second to buffer the video and it buffer it very slowly.
2. The glitch and jitter is still there, same spot for the video and still no change.

PS. I removed the cache file and chunk folder before test.
